rems/config.edn

```diff
@@ -0,0 +1,29 @@
+; SPDX-FileCopyrightText: 2024 Stichting Health-RI
+;
+; SPDX-License-Identifier: Apache-2.0
+
+{:port 3000
+ :public-url "http://localhost:3000/"
```

Comment:
to follow this deployment example, the public url should be `http://rems.local.onemilliongenomes.eu/` instead.

Reply:
adjusted